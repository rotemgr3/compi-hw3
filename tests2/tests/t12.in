//piazza 80 - should be errorMainMissing +
int main (int a) {int x = 5;}