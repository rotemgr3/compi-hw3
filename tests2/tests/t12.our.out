---end scope---
a INT -1
x INT 0
Program has no 'void main()' function
